Assets/02_Student Folders/SaidKrebbers_Assets/Scripts/compress.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class compress : MonoBehaviour
+{
+  private bool depressing = false;
+  private bool pressing = false;
+  private bool pressed = false;
+  private float speed = 0.2f;
+  private float speed2 = 5;
+  private float timer = 1f;
+  private float timer2 = 1f;
+  public Vector3 startPosition;
+  public Vector3 endPosition;
+  public Vector3 weaponEndPosition;
+  private bool activated = false;
+  private bool done = false;
+
+  public Transform button;
+  public Transform weaponDoor;
+  public Collider triggerZone;
+
+
+
+  // Start is called before the first frame update
+  void Start()
+  {
+    pressing =false;
```

Comment:
Unnecesarry if you are already doing that above

Reply:
Doesn't have to be changed but just a small remark if you already gave it a value on creation you dont have to again assign it the same value on Start()

Reply:
That... is true, it is unnecessary (this statement was written during testing)
I do believe however that it is not necessarily wronk.